VolumeClipWithModel/VolumeClipWithModel.py

```diff
@@ -294,15 +313,17 @@ def onClippingMarkupSelect(self, node):
     self.setAndObserveClippingMarkupNode(self.clippingMarkupSelector.currentNode())
 
   def onOutputVolumeSelect(self, node):
-    self.updateApplyButtonState()    
-    
+    self.updateApplyButtonState()
+
   def onApplyButton(self):
     inputVolume = self.inputVolumeSelector.currentNode()
     outputVolume = self.outputVolumeSelector.currentNode()
     clippingModel = self.clippingModelSelector.currentNode()
-    clipOutsideSurface = self.clipOutsideSurfaceCheckBox.checked
-    fillValue = self.fillValueEdit.value
-    self.logic.clipVolumeWithModel(inputVolume, clippingModel, clipOutsideSurface, fillValue, outputVolume)
+    ClipOutsideSurface = self.ClipOutsideSurfaceCheckBox.checked
+    ClipInsideSurface = self.ClipInsideSurfaceCheckBox.checked
+    FillOutsideValue = self.FillOutsideValueEdit.value
+    FillInsideValue = self.FillInsideValueEdit.value

```

Comment:
make capitalization consistent with other parts of the code (local variable names are not capitalized)